src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
```

Comment:
The class `DataConverterImpl` is located in the default package. According to the checklist, it should be located in a corresponding package. Consider moving it to the `service.impl` package.